### PR TITLE
Add draft accessibility statement

### PR DESCRIPTION
**Pull request by nadonomy**
Added a draft accessibility statement. @janogarcia @gsouquet would like review from both of yourself, in particular:

- @gsouquet: What else do you think is missing? Any past learnings or statements we could fold in from working on Element Web?
- @janogarcia: The same. Also, do you have confidence in specific text, contrast ratios etc we should spell out? If so, can you either commit to this branch directly or comment in review and I'll amend?

Resolves https://github.com/vector-im/compound/issues/57

### CODE REVIEW

File: stories/foundations/Accessibility.stories.mdx [hunk 1]
```diff
@@ -4,4 +4,71 @@ import { Meta } from "@storybook/addon-docs";
 
 # Accessibility
 
-...
\ No newline at end of file
+Designing and building with accessibility in mind is crucial as it ensures all individuals, regardless of abilities, can access and use our products.
+
+Moreover, designing for accessibility is beneficial for everyone. For instance, using clear and concise language and east-to-use navigation benefits everyone, regardless of abilities.
```
Comment (gsouquet) [inline stories/foundations/Accessibility.stories.mdx:9]:
```suggestion
Moreover, designing for accessibility is beneficial for everyone. For instance, using clear and concise language and easy-to-use navigation benefits everyone, regardless of abilities.
```

File: stories/foundations/Accessibility.stories.mdx [hunk 2]
```diff
@@ -4,4 +4,71 @@ import { Meta } from "@storybook/addon-docs";
...
+
+This means we support:
+
+* **Accessible labels:** Form controls are built using accessible names and controls as laid out by the [WAI-ARIA specification](https://www.w3.org/TR/wai-aria-1.2/#namecalculation).
+* **Keyboard navigation:** Keyboard navigation support in accordance with [WAI-ARIA authoring practises](https://www.w3.org/TR/wai-aria-practices-1.2/).
+* **Focus management:** Logical focus movement as users interact with elements and the interface changes as a result. Moving focus based on normal actions in any context, for example switching focus to the primary button when showing an alert.
+
+### Disproportionate burden
+
+Although we strive to hold ourselves accountable to these standards, there are limits to how much Compound can solve wider accessibility needs in Element. For example, it's impractical for Compound to:
```
Comment (gsouquet) [inline stories/foundations/Accessibility.stories.mdx:43]: It's fair to say we are not going to cover certain things. But I believe it's important to say that an accessible design system does not translate into an accessible product, but rather helps designers ad engineers make sane decisions that will translate into an accessible product

**Comment by janogarcia** [comment]
@nadonomy We wouldn't need to add such specific details there (minimum contrast ratio for text, minimum contrast ratio for non-text content, etc.), but we do need to mention one obvious thing, the WCAG compliance level, which should be AA. Otherwise it could be anything from A to AAA.

**Comment by janogarcia** [comment]
Typo in `east-to-use navigation`.

**Review by gsouquet — APPROVED** [review]
I like this as a starting point!

We'll probably extend on the testing accessibility in the future, but I don't think I have anything smart to add on that front at this stage.

**Comment by janogarcia** [comment]
Also, if not reviewed already, Gov.uk published a sample accessibility statement with some useful bits we can repurpose.

https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website

**Comment by janogarcia** [comment]
It seems we are publishing this to `Foundations/Accessibility` but we shouldn't mix the scope and contents of an Accessibility Statement vs. the Accessibility Guidelines.

First one is for those using a product/service, while second one would be the specific accessibility practices those authoring or contributing to the product/service should follow.

**Comment by gsouquet** [comment]
@nadonomy Do you consider that this would solve https://github.com/vector-im/compound/issues/57 ?

File: stories/foundations/Accessibility.stories.mdx (hunk 2 above)
Comment (nadonomy) [reply stories/foundations/Accessibility.stories.mdx:43]: Good point. Will iterate and commmit.

File: stories/foundations/Accessibility.stories.mdx [hunk 3]
```diff
@@ -4,4 +4,71 @@ import { Meta } from "@storybook/addon-docs";
...
+
+* Provide live closed captioning for end-to-end encrypted video calls.
```
[the remark this answers is not in the document]
Comment (nadonomy) [reply stories/foundations/Accessibility.stories.mdx]: Ack but prefer to merge and get more sophisticated later.

File: stories/foundations/Accessibility.stories.mdx [hunk 4]
```diff
@@ -4,4 +4,71 @@ import { Meta } from "@storybook/addon-docs";
...
+2. **Operable:** User interface components and navigation must be operable.
+3. **Understandable:** Information and the operation of user interface must be understandable.
+4. **Robust:** Content must be robust enough that it can be interpreted reliably by a wide variety of user agents, including assistive technologies.
+
+While also providing guidance on areas such as:
+
+* Text alternatives for non-text content
+* Keyboard accessibility
+* Colour contrast
+* User-friendly forms
```
[the remark this answers is not in the document]
Comment (nadonomy) [reply stories/foundations/Accessibility.stories.mdx:29]: Thx! Added.

**Comment by nadonomy** [comment]
@gsouquet @janogarcia thanks for the feedback! I think this should be good to merge now.

On the other points not solved by review comment threads or commits:

> Also, if not reviewed already, Gov.uk published a sample accessibility statement with some useful bits we can repurpose.
> → https://www.gov.uk/government/publications/sample-accessibility-statement

@janogarcia This is a helpful resource, thanks! We can use it for inspiration when we iterate further. For now, I just wanted to nail the basics to help align internal stakeholders and communication with customers.

Feel free to add more high signal detail or re-jig based on this in future.

> It seems we are publishing this to `Foundations/Accessibility` but we shouldn't mix the scope and contents of an Accessibility Statement vs. the Accessibility Guidelines.
> 
> First one is for those using a product/service, while second one would be the specific accessibility practices those authoring or contributing to the product/service should follow.

@janogarcia agree! The intent is to put the relevance guidance in the design/develop sections.

> @nadonomy Do you consider that this would solve #57 ?

@gsouquet yep! Good point, adding to the main PR body to auto close.

**Review by janogarcia — APPROVED** [review]

**Comment by janogarcia** [comment]
> The intent is to put the relevance guidance in the design/develop sections.

@nadonomy We might find that having such a hard split for foundational guidelines such as accessibility might not be that practical to maintain and consume. A single resource on accessibility guidelines may be easier both for maintainers and consumers. But yeah, let's decide once we have drafted the contents.

**Pull request merged**